### DISCUSSION TITLE
GitHub Global Campus Approved but problems persist

### DISCUSSION THREAD

**Discussion by JM2K69** [community/community#56745] [GitHub Education]
### Hi everyone,

Hello, my Global Campus application has been approved. Students have access to the Github developperPack but not teachers. Why is this? How can I give them access to these advantages?

Thank for your help.

**Comment by jge162** [comment]
Here is link to GitHub [Education](https://education.github.com/teachers). You can check all offerings there. There are links to verify your academic employment.

**Comment by JM2K69** [comment]
All seems good I dont need to update registration I dont understand why i have non acces.

**Reply by jge162** [reply]
How long ago were you approved?

**Reply by JM2K69** [reply]
In June 2022 and the application say not need to apply again your are approved.

**Reply by jge162** [reply]
I understand now. You’re kinda stuck in between renewal. Might need a Mod to help you. Not sure what your options would be since you are already approved but your account is not enabled.

**Reply by JM2K69** [reply]
Perhaps or even certainly but I have to wait? I opened a support ticket but nothing.

**Reply by jge162** [reply]
Mods can assist to an extent. Queenofcorgis helps a lot. Just gotta wait unfortunately.

**Reply by JM2K69** [reply]
Any update or help please.

**Comment by samus-aran** [comment]
Hi @JM2K69 👋 

What Global Campus Benefits are you unable to obtain as a verified teacher? Teachers do not receive the Student Developer Pack as part of their benefits if that is what you were seeking.

**Reply by JM2K69** [reply]
In campus éducation GitHub yes. I open a ticket et it seems to je resolved
